src/streams/VirgilStreamCipher.ts

```diff
@@ -29,6 +32,12 @@ export class VirgilStreamCipher extends VirgilStreamCipherBase {
 		for (const { identifier, key} of publicKeyArr) {
 			this.seqCipher.addKeyRecipientSafe(identifier, key);
 		}
+
+		if (signature) {
+			const signatureKey = Buffer.from(DATA_SIGNATURE_KEY);
+			const customParams = this.seqCipher.customParams();
+			customParams.setDataSafe(signatureKey, signature);
```

Comment:
Since `signature` is of type `Data`, which can be a string, it must be converted to `Buffer` with `anyToBuffer` function before being passed into `setDataSafe`